CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing
+
+To contribute to this repository, fork it and send pull requests.
+
+## Project structure
+
+TBD
+
+## Run the project
+
+TBD
+```
+
+## Commit messages
+
+TBD
+
+## Pull Request Process
+
+1. Update the [README.md](README.md) or similar documentation with details of changes you wish to make, if applicable.
```

Comment:
?

Reply:
This should say CHANGELOG instead I think. But there's no changelog yet.

Reply:
I'll add to this PR.

Reply:
Then we can create one based on the details in the releases page.

Reply:
I added a changelog file and updated the contributing doc accordingly.